Lib/os.py

```diff
@@ -133,34 +133,3 @@ def getenv(key, default=None):
     The optional second argument can specify an alternate default.
     key, default and the result are str."""
     return environ.get(key, default)
-
-
-def fspath(path):
```

Comment:
Will keeping this funciton as `_fspath` just as cpython help to test new os_fspath works correctly?
Probably we can test 2 functions return same value.

Reply:
We are testing against CPython function